src/Model/ROI.py

```diff
@@ -715,9 +717,63 @@ def transform_rois_contours(axial_rois_contours):
                         sagittal_rois_contours[name][contour[i][0]] = [[]]
                         sagittal_rois_contours[name][contour[i][0]][0]\
                             .append([contour[i][1], slice_ids[slice_id]])
+
+    coronal_rois_contours = convert_coordinates_map_to_polygon_of_rois(
+        coronal_rois_contours)
+    sagittal_rois_contours = convert_coordinates_map_to_polygon_of_rois(
+        sagittal_rois_contours)
     return coronal_rois_contours, sagittal_rois_contours
 
 
+def convert_coordinates_map_to_polygon_of_rois(contours_map):
+    polygon_dict = {}
+    for name, contours_dict in contours_map.items():
+        polygon_dict[name] = {}
+        for slice_id, contour_array in contours_dict.items():
+            roi_array = contour_array[0]
+            hull_list = calculate_concave_hull_of_points(roi_array, alpha=0)
+            polygon_dict[name][slice_id] = hull_list
+    return polygon_dict
+
+
+def calculate_concave_hull_of_points(pixel_coords, alpha):
+    """
+        Return the alpha shape of the highlighted pixels using the alpha
+        entered by the user.
+        :param pixel_coords: the coordinates of the contour pixels
+        :return: List of lists of points ordered to form polygon(s).
+        """
+    # Get all the pixels in the drawing window's list of highlighted
+    # pixels, excluding the removed pixels.
+    target_pixel_coords = [(item[0] + 1, item[1] + 1) for item in
+                           pixel_coords]
+    # Calculate the concave hull of the points.
+    # alpha = 0.95 * alphashape.optimizealpha(points)
```

Comment:
Is this supposed to be commented out?

Reply:
yes. It's from last year's team. I think they planned to generate an "optimal" alpha value to generate polygons from a list of pixels, but since the operation takes too long (I attempted to run it), they commented it out and used user-input alpha value instead. I will add a # TODO comment for future team to look at it.